Makefile

```diff
@@ -19,6 +19,13 @@ build:
 	sh scripts/build-swagger.sh
 	hugo
 
+# muffet - checks for broken links - you have to have hugo running for this to work
+muffet:
+	@if ! curl -s --fail http://localhost:1313 > /dev/null ; then echo "You must start a Hugo local server!"; exit 1; fi
+	@mkdir -p $(CURDIR)/output/muffet
+	@ if [ ! -f $(CURDIR)/output/muffet/bin/muffet ] ; then GOPATH=$(CURDIR)/output/muffet go get -u github.com/raviqqe/muffet; fi
+	@$(CURDIR)/output/muffet/bin/muffet http://localhost:1313/
```

Comment:
Still working through this a little more on my side, but it looks like adding these options take care of a number of the anchor problems:

`--exclude \#\!\(forum\|msg\)\/kiali-\(dev\|users\)\/? --exclude=https?:\/\/localhost:20001\/?`

Reply:
I added those and one more additional one for that documentation/developer-api which appears to be some dynamically generated doc